README.md

```diff
@@ -50,34 +48,6 @@ optional arguments:
   --verbose, -v         Run verbosely
 ```
 
-### Providing credentials via environment variables
```

Comment:
It would be great if you could describe, that username and password are requested during first run.